pkg/netdevice/netResourcePool.go

```diff
@@ -80,3 +84,42 @@ func (rp *netResourcePool) GetDeviceSpecs(deviceIDs []string) []*pluginapi.Devic
 	}
 	return devSpecs
 }
+
+// StoreDeviceInfoFile stores the Device Info files according to the
+//  k8snetworkplumbingwg/device-info-spec
+func (rp *netResourcePool) StoreDeviceInfoFile(resourceNamePrefix string) error {
+	for id, dev := range rp.GetDevicePool() {
+		netDev, ok := dev.(types.PciNetDevice)
+		if !ok {
+			return fmt.Errorf("StoreDeviceInfoFile: Only pciNetDevices are supported")
+		}
+		devInfo := nettypes.DeviceInfo{
+			Type:    "pci",
```

Comment:
should this also use the const defined in net-attach-def-client for "pci" Type?

Reply:
Done. Thanks